bodhi/server/util.py

```diff
@@ -297,9 +298,23 @@ def hostname(context=None):
 
 
 def markup(context, text):
-    return markdown.markdown(text, safe_mode="replace",
-                             html_replacement_text="--RAW HTML NOT ALLOWED--",
-                             extensions=['markdown.extensions.fenced_code'])
+    """
+    Return HTML from a markdown string
+
+    Args:
+        context:
```

Comment:
ah! forgot that this method was used in the templates too, where context is passed automatically!

do you think we should just leave it as context, since all the others use it?